ivy/functional/frontends/paddle/nn/functional/vision.py

```diff
@@ -170,3 +170,105 @@ def pixel_shuffle(x, upscale_factor, data_format="NCHW"):
     return ivy.reshape(
         ivy.permute_dims(input_reshaped, (0, 1, 4, 2, 5, 3)), (b, oh, ow, oc)
     )
+@to_ivy_arrays_and_back
```

Comment:
We need two lines of whitespace between functions

Reply:
Thank you @Sam-Armstrong for the corrections here